app/assets/javascripts/application.js

```diff
@@ -24,4 +24,6 @@
 
 $('body').on('click', '[disabled], .disabled', function (e) {
   e.preventDefault();
-});
\ No newline at end of file
+});
+
+const PUBLIC_CLOUDS = ['azure', 'ec2', 'gce'];
```

Comment:
Please change it to `var` because phantomjs doesn't support this and it will fail.

Reply:
The specs pass.